model/vm.rb

```diff
@@ -51,6 +52,10 @@ def ip4
     assigned_vm_address&.ip
   end
 
+  def runtime_token
+    JWT.encode({sub: ubid, iat: Time.now.to_i}, Config.clover_runtime_token_secret, "HS256")
```

Comment:
If we add future use-cases, we could consider extending the subject (or use a different claim) to embed the use-case this claim is good for. For example, a claim with subject `vmabc123...-runnercache` would only be good for that use-case but not for anything else.

Reply:
Indeed, we can also include the scope value in the payload for potential future use cases.